### DISCUSSION TITLE
Can only Administrators create Read Lists?

### DISCUSSION THREAD

**Discussion by madsciencepro** [gotson/komga#1776] [Q&A] [answered]
I've got a couple friends who I setup with regular User accounts. We discovered that they don't have the option to create Read Lists. Is there something I need to enable to fix that?

**Comment by gotson** [comment, marked as answer]
only admins can create read lists and collections.

**Reply by madsciencepro** [reply]
Any plans to add that feature to Users in the future? Thanks!
